modules/rules_toolchain/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "homepage": "https://gitlab.arm.com/bazel/rules_toolchain",
+    "repository": [
+        "https://gitlab.arm.com/bazel/rules_toolchain"
```

Comment:
This is the first gitlab hosted repo. We have some spots that are hardcoded for GitHub both here and in BCR-ui repo that will need to be conditional/generalized 😅

Reply:
I'll have a look at the validation script to see what I can do.

Reply:
#1284 sorts that out. Needs to be a separate PR as cannot change them when adding a new module.